editions/tw5.com/tiddlers/filters/syntax/Map Filter Run Prefix (Examples).tid

```diff
@@ -14,3 +14,28 @@ Replace the input titles with the caption field if it exists, otherwise preserve
 For each title in a shopping list, calculate the total cost of purchasing each item:
 
 <<.operator-example 2 "[tag[shopping]] :map[get[quantity]else[0]multiply{!!price}]">>
+
+!! Comparison between `:map` and `:and`/`+` filter run prefixes
```

Comment:
We should consider the `+` name deprecated. I think it is enough to mention once in this tiddler that they are the same and then always use `:and` in both the descriptions and the examples.

Reply:
Sure, I made that change.